client/me/account-password/index.jsx

```diff
@@ -158,23 +164,27 @@ const AccountPassword = React.createClass( {
 					<FormButton
 						disabled={ this.state.pendingValidation || this.props.accountPasswordData.passwordValidationFailed() }
 						onClick={ this.recordClickEvent( 'Save Password Button' ) }>
-						{ this.state.savingPassword ? this.translate( 'Saving…' ) : this.translate( 'Save Password' ) }
+						{ this.state.savingPassword ? translate( 'Saving…' ) : translate( 'Save Password' ) }
 					</FormButton>
 
 					<FormButton
 						className="button"
 						isPrimary={ false }
 						onClick={ this.recordClickEvent( 'Generate Strong Password Button', this.generateStrongPassword ) }
 						type="button">
-						{ this.translate( 'Generate strong password' ) }
+						{ translate( 'Generate strong password' ) }
 					</FormButton>
 				</FormButtonsBar>
 			</form>
 		);
 	}
 } );
 
-export default connect(
-	null,
-	dispatch => bindActionCreators( { errorNotice }, dispatch )
-)( protectForm( AccountPassword ) );
+export default compose(
```

Comment:
Thanks @gziolo!
After some feedback I'm hoping to open some docs/guides around it's usage... possibly even an lint rule too if I can figure that out!  
I've opened an issue for discussion over at: #10138 
😀